app/assets/stylesheets/govuk_publishing_components/components/_layout-super-navigation-header.scss

```
@@ -128,8 +130,8 @@ $search-icon-size: 20px;
 
     &:before,
     &:after {
-      $width: 1.2rem;
-      $height: .2rem;
+      $width: 20px;
```

Can we use `govuk-spacing` here?

```suggestion
      $width: govuk-spacing(4);
```

---

Sure thing - done: 

https://github.com/alphagov/govuk_publishing_components/blob/9a87c5d73cca8f1070f647452db00abb26b3a254/app/assets/stylesheets/govuk_publishing_components/components/_layout-super-navigation-header.scss#L133